udemy-reviews-aggregator/src/main/java/com/github/simplesteph/kafka/streams/udemy/aggregator/AggregatorMain.java

```diff
@@ -42,40 +49,44 @@ private void start() {
 
     private Properties getKafkaStreamsConfig() {
         Properties config = new Properties();
-        config.put(StreamsConfig.APPLICATION_ID_CONFIG, appConfig.getApplicationId());
-        config.put(StreamsConfig.BOOTSTRAP_SERVERS_CONFIG, appConfig.getBootstrapServers());
-        config.put(ConsumerConfig.AUTO_OFFSET_RESET_CONFIG, "earliest");
+        config.put(APPLICATION_ID_CONFIG, appConfig.getApplicationId());
```

Comment:
Can we please revert those?

Reply:
Sure I can

Reply:
Yes I will revert those code style changes today.